### ISSUE TITLE
False positives from no-with-statement rule

### ISSUE THREAD

**Issue by chrismbarr** [Microsoft/tslint-microsoft-contrib#47]
When I enable the `no-with-statement` rule, it gives me **a lot** of false positives.  I have the following test code:
```typescript
module MyApp {
    "use strict";

    export class TestClass {
		test = "test";
    }
}
```

And running TSLint against that produces the following output
```
[13:37:03] Using gulpfile C:\<MY_FILE_PATH>\Gulpfile.js
[13:37:03] Starting 'TSLint:Newer'...
[13:37:03] [gulp-tslint] error test-file.ts[2, 5]: Forbidden with statement
[13:37:03] Finished 'TSLint:Newer' after 545 ms
Process terminated with code 0.
```

yeah... so it seems to think like line 2 of that test code above contains a `with` statement... I have no idea why.

I am running this in **Visual Studio 2015** with **`gulp` v3.9.0** and **`gulp-tslint` v3.3.0**

**Comment by HamletDRC** [comment]
Thanks for taking the time to file this. 

I will take a look tonight. I suspect that differing versions of TypeScript is to blame.

**Comment by chrismbarr** [comment]
Ah yes, I suppose that would be important information to know as well!  I am using TypeScript v1.6.3.0

**Comment by HamletDRC** [comment]
I upgraded our tslint dependency to 2.5.1 and there are a bunch of test failures. Certainly this issue is that we do not yet support 2.5.1. I created Issue #49 to track this.

**Comment by HamletDRC** [comment]
OK, I fixed issue #49. This issue is fixed with it. We'll try to push a new version by Monday. 

I really appreciate your patience and willingness to give us a chance.
